### PR TITLE
Infrastructure update.

### PR DESCRIPTION
**Pull request by koresar**
Use latest ESLint, Babel, Rollup.
Rename distr. bundle files to avoid common confusions.

### CODE REVIEW

**Comment by koresar** [comment]
Oops! ESLint v3 dropped support of node 0.12.
Should we also drop support of node 0.12?

**Comment by FredyC** [comment]
I thought we already agreed on dropping support for 0.12. You even made issue for it :) https://github.com/stampit-org/stampit/issues/224

**Comment by koresar** [comment]
I admire your memory! Dropping it now (because I dropped only the 0.10). :)

**Comment by FredyC** [comment]
It took me a while to find it, I only vaguely knew, we talked about it.

File: .eslintrc [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+  "extends": "airbnb-base",
   "rules": {
     "object-curly-spacing": [2, "never"],
     "no-param-reassign": 0,
     "comma-dangle": 0,
     "no-confusing-arrow": 0,
     "prefer-rest-params": 0,
-    "no-use-before-define": 1
+    "no-use-before-define": 1,
+    "no-plusplus": 0
```
Comment (FredyC) [inline .eslintrc]: Scratch that only occurrence, it's also in some for loops. But still I don't like that operator :)

**Pull request merged**